src/index.js

```diff
@@ -32,6 +32,7 @@ const App = () => {
             <Route exact path="/admin" component={requireAuth(AdminPage)} />
             <Route exact path="/authenticate-email" component={AuthPage} />
             <Route exact path="/forgot-password" component={PasswordPage} />
+            <Route exact path="/reset-password" component={PasswordUpdate} />
```

Comment:
ALPHABETICAL ORDER

Reply:
also, email link is for `/resetPassword` (see [here](https://github.com/acmucsd/membership-portal/blob/master/app/email/index.js#L21))